src/BaseFirestoreRepository.spec.ts

```diff
@@ -237,6 +284,55 @@ describe('BaseFirestoreRepository', () => {
       const updatedBand = await bandRepository.update(band);
       expect(band.name).to.equal(updatedBand.name);
     });
+
+    it('must not validate if the validate config property is false', async () => {
+      initialize(firestore, { validate: false });
+
+      bandRepository = new BandRepository('bands');
+
+      const band = await bandRepository.findById('porcupine-tree');
+
+      band.contactEmail = 'test@email.com';
```

Comment:
Shouldn't this be an invalid email?

Reply:
Yeah you're right. This should fix your comment about the missing unit test too